### PR TITLE
Markdown linting README.md

### LINKED ISSUE #306: README.md fails linting
I'll have a PR shortly:

```bash
$ mdl README.md
README.md:7: MD012 Multiple consecutive blank lines
README.md:3: MD013 Line length
README.md:4: MD013 Line length
README.md:20: MD013 Line length
README.md:23: MD013 Line length
README.md:32: MD013 Line length
README.md:37: MD013 Line length
README.md:39: MD013 Line length
README.md:40: MD013 Line length
README.md:41: MD013 Line length
README.md:42: MD013 Line length
README.md:43: MD013 Line length
README.md:45: MD013 Line length
README.md:46: MD013 Line length
README.md:47: MD013 Line length
README.md:49: MD013 Line length
README.md:50: MD013 Line length
README.md:51: MD013 Line length
README.md:52: MD013 Line length
README.md:53: MD013 Line length
README.md:55: MD013 Line length
README.md:56: MD013 Line length
README.md:57: MD013 Line length
README.md:58: MD013 Line length
README.md:60: MD013 Line length
README.md:61: MD013 Line length
README.md:62: MD013 Line length
README.md:63: MD013 Line length
README.md:64: MD013 Line length
README.md:65: MD013 Line length
README.md:73: MD013 Line length
README.md:74: MD013 Line length
README.md:83: MD013 Line length
README.md:104: MD013 Line length
README.md:106: MD013 Line length
README.md:132: MD013 Line length
README.md:1: MD022 Headers should be surrounded by blank lines
README.md:12: MD022 Headers should be surrounded by blank lines
README.md:22: MD022 Headers should be surrounded by blank lines
README.md:25: MD022 Headers should be surrounded by blank lines
README.md:31: MD022 Headers should be surrounded by blank lines
README.md:67: MD022 Headers should be surrounded by blank lines
README.md:93: MD022 Headers should be surrounded by blank lines
README.md:118: MD022 Headers should be surrounded by blank lines
README.md:129: MD022 Headers should be surrounded by blank lines
README.md:81: MD026 Trailing punctuation in header
README.md:19: MD028 Blank line inside blockquote
README.md:26: MD031 Fenced code blocks should be surrounded by blank lines
README.md:69: MD031 Fenced code blocks should be surrounded by blank lines
README.md:77: MD031 Fenced code blocks should be surrounded by blank lines
README.md:107: MD031 Fenced code blocks should be surrounded by blank lines
README.md:112: MD031 Fenced code blocks should be surrounded by blank lines
README.md:13: MD032 Lists should be surrounded by blank lines
README.md:97: MD032 Lists should be surrounded by blank lines
README.md:120: MD032 Lists should be surrounded by blank lines
README.md:131: MD032 Lists should be surrounded by blank lines

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

### PR DESCRIPTION
**Pull request by mamercad**
Fixes #306; ~adding `markdownlint` to CI as well~.

### CODE REVIEW

**Review by armab — CHANGES_REQUESTED** [review]
Thanks for the PR!

While the Markdown linting might look like a good enhancement at first, unlike `.yaml` linter where issues should be actionable or `.rst` linter for docs.stackstorm.com where the issue might lead to a broken template, or `ansible-lint` which is based on the best practices, Markdown linter provides just a little value while bringing lots of noise for a single README.md.

File: README.md [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
-With over [50+ integrations](https://github.com/StackStorm/st2contrib/tree/master/packs) like GitHub, Docker, Nagios, NewRelic, AWS, Ansible it allows you to wire together your existing infrastructure into complex Workflows with auto-remediation and many more.
-Aka IFTTT orchestration for Ops.
 
+> [StackStorm](http://stackstorm.com/) is event-driven automation platform
+> written in Python.
+
+With over [50+ integrations](https://github.com/StackStorm/st2contrib/tree/master/packs)
+like GitHub, Docker, Nagios, NewRelic, AWS, Ansible it allows you to wire
+together your existing infrastructure into complex Workflows with
+auto-remediation and many more.  Aka IFTTT orchestration for Ops.
```
[the remark this answers is not in the document]
Comment (mamercad) [reply README.md:11]: Cool, I'll remove the CI and fix the problems in the README when I get a few.

**Comment by mamercad** [comment]
Down to only line length warnings:

```bash
❯ mdl README.md 
README.md:51: MD013 Line length
README.md:53: MD013 Line length
README.md:54: MD013 Line length
README.md:55: MD013 Line length
README.md:56: MD013 Line length
README.md:57: MD013 Line length
README.md:59: MD013 Line length
README.md:60: MD013 Line length
README.md:61: MD013 Line length
README.md:63: MD013 Line length
README.md:64: MD013 Line length
README.md:65: MD013 Line length
README.md:66: MD013 Line length
README.md:67: MD013 Line length
README.md:69: MD013 Line length
README.md:70: MD013 Line length
README.md:71: MD013 Line length
README.md:72: MD013 Line length
README.md:74: MD013 Line length
README.md:75: MD013 Line length
README.md:76: MD013 Line length
README.md:77: MD013 Line length
README.md:78: MD013 Line length
README.md:79: MD013 Line length

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

**Comment by armab** [comment]
Hi @mamercad, we don't enforce the line length limit for Markdown files.
Is there any specific issue you're having with rendering the README.md?
Please include those practical fixes and enhancements only.

To make the build green, you can update your working branch with the most recent upstream master.

**Comment by mamercad** [comment]
> Hi @mamercad, we don't enforce the line length limit for Markdown files. Is there any specific issue you're having with rendering the README.md? Please include those practical fixes and enhancements only.
> 
> To make the build green, you can update your working branch with the most recent upstream master.

The linting problems were visual noise in my IDE, that's all. I'll rebase shortly.

**Comment by mamercad** [comment]
If you'd like me to undo line length, just let me know.

**Comment by armab** [comment]
@mamercad Yes, please.

**Review by armab — APPROVED** [review]
Thanks :+1:

**Pull request merged**